get-started/setup-an-ide.md

```diff
@@ -1,7 +1,88 @@
 # Setup an IDE
 
-* vs code
-* goland
-* vim \(optional\)
-* emacs \(optional\)
+## vs code
+
+## goland
+
+You can use GoLand to easily run or debug TiDB in many situations, .
+
+0. make sure your `go` and `dlv` work well.
```

Comment:
We can capitalize the first word, and provide a script for evaluate that `go` and `dlv` work well. BTW, what is `dlv`?

Reply:
The full name of dlv is [delve](https://github.com/go-delve/delve), which is the most popular debugging tool of golang, and used by GoLand debugging.

In my assumption, the previous chapters will tell users how to make these tools work well(x).